config/initializers/omniauth.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+# use OmniAuth::Builder do
+  provider :github, ENV['GITHUB_KEY'], ENV['GITHUB_SECRET']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.